tests/benchmark/conftest.py

```diff
@@ -7,6 +7,17 @@
 
 def pytest_collection_modifyitems(config, items):
     """Add the `benchmark` marker to all tests under `./tests/benchmark`."""
+    marker_expr = config.getoption("-m", default="")
```

Comment:
I think we could even use `--gas-benchmark-values` introduced in #1895, I'm not sure if we can access that option from here but that would be ideal IMO.